types/adm-zip/index.d.ts

```diff
@@ -364,147 +364,147 @@ declare namespace AdmZip {
         extraLen: number;
     }
 
-    enum Constants {
+    interface Constants {
```

Comment:
Sorry, I blanked.  You probably actually want 
```ts
declare const Constants : {
    /* The local file header */
    LOCHDR: 30; // LOC header size
    LOCSIG: 0x04034b50; // "PK\003\004"
    LOCVER: 4; // version needed to extract
    //...
}
```
since consumers will only use it for values, not as a type.

Reply:
NVM, I found a way of making it work. I've moved the declaration of the const out to `/util` and imported it in index where it is used as a type in `method`.